src/html/transfers.html

```diff
@@ -448,7 +455,9 @@ <h3>${window.utils.formatLargeNum(transfer.amount, transfer.decimals)} ${transfe
               transfer.callToAction
             }</button>`
           )}
-          ${window.dom.toString(transfer.status === 'completed' &&
+          ${window.dom.toString(
+            (transfer.status === 'completed' || !transfer.completedStep ||
+             (transfer.completedStep === 'approve-natural-erc20-to-nep141' && transfer.status === 'action-needed')) &&
```

Comment:
I'm not sure I understand the intent of this change, but the `delete.svg` icon uses a default color that matches the `var(--fg)` color of `.transfer.completed`. If we render it inline by moving it to the `html` folder and using `include` instead of `img`, then it will Just Work and match the other colors in the transfers perfectly. If we want to avoid that, then it probably makes sense to make the default color some shade of gray or black, rather than the current dark green.